src/DFC_BusinessOntology.owl

```diff
@@ -8129,6 +8773,16 @@ Any claims of a Product.
         <IRI>#endDate</IRI>
         <Literal xml:lang="en">end date</Literal>
     </AnnotationAssertion>
+    <AnnotationAssertion>
+        <AnnotationProperty abbreviatedIRI="rdfs:label"/>
+        <IRI>#endsAt</IRI>
+        <Literal xml:lang="en">ends at</Literal>
```

Comment:
What ends? Why? 

This is on Shipment, so it's the destination of the Shipment, yes ?

Reply:
Yes, the shipment should have a departure and a destination